deploifai/api/__init__.py

```diff
@@ -336,7 +354,13 @@ def create_project(self, project_name: str):
             headers=self.headers,
         )
 
+        if r.status_code == 200:
+            click.secho(f"Successfully created new project named {project_name}.", fg="green")
+        else:
+            click.secho(f"An error has occured while trying to create a new project. Please try again.", fg="red")
```

Comment:
These `click.secho` message prints should not be in the methods of the `DeploifaiAPI`

Reply:
in this case, should I do an exception handling for each api call's response, like when doing sth like `create_mutation_data = r.json()`

`return create_mutation_data["data"]["createProject"]`?

Reply:
raise error in api/__init__.py and handle it in the command call back